conda/libmesh-vtk/build.sh

```diff
@@ -30,7 +30,8 @@ cmake .. -G "Ninja" \
     -DVTK_GROUP_ENABLE_Web:STRING=NO \
     -DCMAKE_OSX_SYSROOT=${CONDA_BUILD_SYSROOT}
 
-ninja install -v
+CORES=$(echo "${CPU_COUNT:-2} / 2" | bc)
+ninja install -v -j $CORES
```

Comment:
I found while building on my machine, it is destroyed by oversubscribing. I think Conda treats threads as cores.

Reply:
In favor of this, as conda build has definitely used more than I thought it should in the past. 👍🏻